modules/xml/xml.c

```diff
@@ -230,8 +234,9 @@ xml_parser_process(LogParser *s, LogMessage **pmsg,
   return TRUE;
 
 err:
-  msg_error("xml: error",
-            evt_tag_str("str", error->message));
+  msg_error("xml-parser failed",
+            evt_tag_str("error", error->message),
+	          evt_tag_int("forwarding message to next pipe element", self->forward_invalid));
```

Comment:
evt_tags should not contain spaces. evt_tag_int("forward_invalid", self->forward_invalid); would be enough.

Reply:
Thanks, will fix it